### ISSUE TITLE
Founding Fathers automatically create peace

### ISSUE THREAD

**Issue by Drunk-Monk** [We-the-People-civ4col-mod/Mod#298]
I think John Jay and other Founding Fathers should offer you a choice about peace rather than automatically establishing peace, as often it is not in your interest to do so, and the Founding Fathers are just meant to provide a benefit.

**Comment by raystuttgart** [comment]
Puh, that is most likely not implemented that fast and I think we should try to release soon again.
Generally I agree though that it is something worth considering and improving.

Also I am not sure if these XML changes would affect current savegames or not
Nightinggale can surely answer that question though.

Also that might have side effects - even if they are small.
Maybe it is easier to concentrate on easier fixes for now.

In Release 2.8 we will most likely have more freedom to change things.

**Comment by Nightinggale** [comment]
I don't think it will affect savegames, but it's a good point. It's something extra, which should be verified. I too am not certain we should do this for 2.7.2, mainly because I want to actually finish 2.7.2.
